packages/location-management/src/components/LocationUnitList/Table.tsx

```diff
@@ -38,32 +37,22 @@ const Table: React.FC<Props> = (props: Props) => {
       width: '10%',
       // eslint-disable-next-line react/display-name
       render: (value: boolean, record: TableData) => (
-        <span className="d-flex justify-content-end align-items-center">
+        <span className="d-flex justify-content-end align-items-center Actions">
           <Link to={URL_LOCATION_UNIT_EDIT + '/' + record.id}>
             <Button type="link" className="m-0 p-1">
               Edit
             </Button>
           </Link>
           <Divider type="vertical" />
-          <Dropdown
-            overlay={
-              <Menu className="menu">
-                <Menu.Item
-                  className="viewdetails"
-                  onClick={() => {
-                    if (onViewDetails) onViewDetails(record);
-                  }}
-                >
-                  {lang.VIEW_DETAILS}
-                </Menu.Item>
-              </Menu>
-            }
-            placement="bottomLeft"
-            arrow
-            trigger={['click']}
+          <Button
+            type="link"
+            className="m-0 p-1"
+            onClick={() => {
+              if (onViewDetails) onViewDetails(record);
```

Comment:
Is there any other state?

Reply:
no there isn't if the event is available it just triggers it else it doesn't

Reply:
What about conditional rendering the button instead of having it do nothing when condition fails?

Reply:
Actually I'm seeing it's always passed to the table component? 
<img width="478" alt="Screenshot 2021-04-06 at 12 17 48" src="https://user-images.githubusercontent.com/10197807/113688629-2ac84a80-96d2-11eb-8b64-f7573869d6f2.png">

Reply:
For now it is, Sure lets just leave it as it is this table component in all packages will soon be replaced by TableLayout (#526) the one i am working on